contrib/readdstar.py

```diff
@@ -141,23 +145,24 @@ def readdstar(getline):
     """
     A = None
     aut_type = None
-    comments = []
-
-    last_state = -1  # -1 indicates unset
+    comments = list()
+    last_state = -1
+        # -1 indicates unset
     try:
         while True:
             line = getline()
             if len(line) == 0:
-                break  # file.readline() returns empty string at EOF
+                break
+                    # file.readline() returns empty string at EOF
             parts = line.split()
             if len(parts) == 0:
-                continue  # Ignore blank lines
-
-            if not parts[0].endswith(":") and len(parts) == 3:  # id line
+                continue
+                    # Ignore blank lines
```

Comment:
Is there a style guide that recommends this pattern of putting comments on the next line with more indentation? I do not see how it is an improvement, but I am willing to try it.